ar/publish.css

```diff
@@ -10,15 +10,16 @@ pre {
 	padding: 32px 18px 0 0;
 }
 .site-body-left-column-site-name {
-	padding: 4px 0 32px 32px;
+	/* padding: 4px 0 32px 32px; */
+	display: none;
```

Comment:
why do we want to hide the site title?

Reply:
If we replace it with the svg, we can hide it.